planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module_data.hpp

```diff
@@ -242,6 +265,7 @@ struct DebugData
 {
   std::shared_ptr<lanelet::ConstLanelets> expanded_lanelets;
   std::shared_ptr<lanelet::ConstLanelets> current_lanelets;
+  std::shared_ptr<lanelet::ConstLineStrings3d> linestring_wrt_overhang;
```

Comment:
@zulfaqar-azmi-t4 
can you make this name more clear? 
for example linestring_to_farthest_overhang

Reply:
is `farthest_linestring_from_overhang` is okay?

Reply:
good

Reply:
I'm on it.